### PR TITLE
feat: Added feature to swipe back from SkillsActivity to ChatActivity

### LINKED ISSUE #2185: Gesture detector to swipe from skills to chat activity
**Describe the feature you'd like**
Swipe feature is there to swipe from chat to skills but not from skills to chat. It should be vice versa

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by Bhavnaharitsa**
Fixes #2185 
Changes: Added changes in SkillsActivity to swipe back to ChatActivity using gesture detector.

### CODE REVIEW

File: app/src/main/java/org/fossasia/susi/ai/skills/SkillsActivity.kt [hunk 1]
```diff
@@ -4,11 +4,10 @@ import android.content.Context
 import android.content.Intent
 import android.os.Bundle
 import android.support.v4.content.ContextCompat
+import android.support.v4.view.GestureDetectorCompat
 import android.support.v7.app.AlertDialog
 import android.support.v7.app.AppCompatActivity
-import android.view.KeyEvent
-import android.view.Menu
-import android.view.MenuItem
+import android.view.*
```
Comment (atm1504) [inline app/src/main/java/org/fossasia/susi/ai/skills/SkillsActivity.kt]: Don't use wildcard imports.

File: app/src/main/java/org/fossasia/susi/ai/skills/SkillsActivity.kt [hunk 2]
```diff
@@ -328,4 +328,30 @@ class SkillsActivity : AppCompatActivity(), SkillFragmentCallback {
...
+
+    //Inner class for handling the gestures
+    internal inner class CustomGestureListener : GestureDetector.SimpleOnGestureListener() {
+        override fun onFling(
+                event1: MotionEvent,
+                event2: MotionEvent,
+                velocityX: Float,
+                velocityY: Float
+        ): Boolean {
+            val SWIPE = event2.getX() - event1.getX()
```
Comment (atm1504) [inline app/src/main/java/org/fossasia/susi/ai/skills/SkillsActivity.kt]: Wrong way of declaring variable name. Use proper naming convention.

**Comment by atm1504** [comment]
@Bhavnaharitsa do provide screenshot to show the code is working. Because, simply writing codes, don’t make the gesture to work. There is 90% chance that the code won't work. So please provide screenshot to show it is working. Thank you :)

**Comment by Bhavnaharitsa** [comment]
> @Bhavnaharitsa do provide screenshot to show the code is working. Because, simply writing codes, don’t make the gesture to work. There is 90% chance that the code won't work. So please provide screenshot to show it is working. Thank you :)

Yeah, sure!

**Comment by atm1504** [comment]
@iamareebjamal @batbrain7 @arundhati24 this code is not working. I have tested it in my mobile Redmi Note 5 Pro. So, I think you can close it. 
I am taking up this issue.

**Pull request closed without merging**